pydicom/filewriter.py

```diff
@@ -460,7 +475,7 @@ def write_dataset(fp, dataset, parent_encoding=default_encoding):
         with tag_in_exception(tag):
             # write_data_element(fp, dataset.get_item(tag), dataset_encoding)
             # XXX for writing raw tags without converting to DataElement
```

Comment:
Looks like these comments can be removed now.

Reply:
could have sworn I removed them...